### ISSUE TITLE
bug(common/web): common/web/types unit test failures

### ISSUE THREAD

**Issue by jahorton** [keymanapp/keyman#8419]
### Describe the bug

With the current state of `master` (https://github.com/keymanapp/keyman/commit/4c68bc1decde6b5ec6ca9be4cbec6496148edd62), the following sequence reports 28 failing unit tests from `common/web/types`:

```
> git clean -fx

> common/web/keyman-version/build.sh build

> common/web/types/build.sh build test
```

I ran into this while working on #8418.

### Reproduce the bug

See above.

### Expected behavior

Unit tests should pass.

**Comment by srl295** [comment]
maybe broke in #8320

**Comment by srl295** [comment]
was #7967 inadvertently closed @mcdurdin ?

**Comment by mcdurdin** [comment]
> was #7967 inadvertently closed @mcdurdin ?

nope, that was fixed in #8131 but just we re-broke the tests since I think.

**Comment by mcdurdin** [comment]
See #8431 for CI
